Library/Formula/apous.rb

```diff
@@ -0,0 +1,23 @@
+class Apous < Formula
+  desc "Apous is a simple tool that allows for easier authoring of Swift scripts."
+  homepage "https://github.com/owensd/apous"
+  url "https://github.com/owensd/apous/archive/v0.2.1.tar.gz"
+  sha256 "d2a283bd328c7dae117de6078448e87a3f3b960876608dd238f5b87f698a10bc"
+
+  depends_on :xcode => ["7.0", :build]
```

Comment:
Heh, well spotted. 7.0 is still beta. You can run it on 10.10 though, if you super want to.